src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -1445,8 +1445,6 @@ End Class
                 ]]>,
```

Comment:
We should clone this test to exercise this scenario against VB14 and the latest (VB15?) language version.